app/web_framework/cluster_base_handler.py

```diff
@@ -11,6 +12,12 @@
 from app.util.session_id import SessionId
 from app.web_framework.api_version_handler import APIVersionHandler
 
+pagination_constants = {
+    'DEFAULT_OFFSET': 0,
+    'DEFAULT_LIMIT': 20,
+    'MAX_LIMIT': 200,
+}
```

Comment:
So are you saying when we do something like

```python
def setUp(self):
    super().setUp()
    Configuration['timings_directory'] = self._TIMINGS_DIR_SYS_PATH
```

`Configuration['timings_directory']` is changed in this particular class test but it gets reset for all the others?